build-locally.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+#
+# This file has been generated by conda-smithy in order to build the recipe
```

Comment:
This file here probably hasn't been directly generated by conda-smithy but copied from a conda-smithy generated repo. Should we mention this somewhere?

Reply:
This file is an exact copy from conda-smithy (i.e. no changes have been made). I'm not sure if it's worth it to just change the comment.